dependencies/src/main/java/com/google/cloud/tools/opensource/dependencies/DependencyGraphBuilder.java

```diff
@@ -49,8 +49,33 @@
 import org.eclipse.aether.util.graph.visitor.PathRecordingDependencyVisitor;
 
 /**
- * Based on the <a href="https://maven.apache.org/resolver/index.html">Apache Maven Artifact
- * Resolver</a> (formerly known as Eclipse Aether).
+ * This class builds dependency graphs for Maven artifacts. The nodes in the graph are Maven
+ * artifacts and its edges are dependencies from an artifact to another.
```

Comment:
artifacts don't have dependencies. A dependency contains an artifact (usually) but is not an artifact.

Reply:
In Maven's `Dependency` and `Artifact` classes, that's right. However, when drawing our graph we don't need to follow that. I felt it's natural to recognize artifacts as nodes and dependencies as edges.